ts/components/ui/utils/baseListItems.tsx

```diff
@@ -0,0 +1,42 @@
+import * as React from "react";
+import { Pressable } from "react-native";
+import Animated from "react-native-reanimated";
+import { WithTestID } from "../../../types/WithTestID";
+import { IOListItemStyles } from "../../core/variables/IOStyles";
+import { useListItemSpringAnimation } from "./hooks";
+
+export type PressableBaseProps = WithTestID<{
+  accessibilityLabel?: string;
+  onPress?: () => void;
+}>;
+export const PressableListItemBase = ({
```

Comment:
IMO, this is not an util but a component. This could go in its own file under `components/ui`, e.g. `PressableListItemBase.tsx`. What do you think?